bin/cbg-process-server

```diff
@@ -24,6 +24,25 @@ COMMANDS:
     push \t Push Process Server Docker image to the Docker Hub
 """
 
+function waitForProcessServerToStart {
```

Comment:
Could you put this waiting block into cbg-integration? cbg-process-server doesn't necessary need to wait there but cbg-integration does. Imagine how would you want it behave when you only start the Process Server without Integration Routes.